inference-engine/src/gna_plugin/gna_model_serial.cpp

```diff
@@ -311,9 +331,19 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
 
     writeBits(header, os);
 
+    for (auto &name : inputNames) {
+        writeBits(static_cast<uint32_t>(sizeof(name.size() + 1)), os);
+        writeBits(strlen(name.c_str()) + 1, os);
```

Comment:
`strlen` return `size_t` like `std::vector::size`.

Reply:
I asked for `strlen` as better suited for 'c_str()' IMO and its not time-perf critical part
see: https://stackoverflow.com/questions/29302073/difference-between-strlenstr-c-str-and-str-length-for-stdstring